aiida_wannier90/calculations.py

```diff
@@ -176,9 +172,9 @@ def prepare_for_submission(self, folder):
             param_dict.update({'postproc_setup': True})
 
         try:
-            local_input_folder = self.inputs.local_input_folder
+            local_input_folder = self.inputs.local_input_folder  #pylint: disable=unused-variable
```

Comment:
It seems to me this variable actually isn't used in the rest of the function.

Reply:
Concerning this point, we had a discussion with @giovannipizzi so I was not sure if we want to delete it or not.

Reply:
It looks to me like the statement doesn't have any effect -- further down the ``local_input_folder`` is again obtained directly from ``self.inputs``.